scripts/variantstore/README.md

```diff
@@ -1,9 +1,31 @@
-
-
+# Running the Genome Variant Store (GVS) Pipeline for AoU
+
+## Steps
+1. If this is the first time running the GVS pipeline in a particular Google billing project, create a support ticket for the BigQuery team that includes "enabling cluster metadata pruning support for the BQ Read API."
+2. This list assumes you are starting off with a Terra workspace that contains the re-blocked gVCFs for your callset as "sample sets" in the "Data" tab.  For workflows, you will be following the steps outlined in the [GVS Quickstart workspace](https://app.terra.bio/#workspaces/broad-dsde-firecloud-billing/Genomic%20Variant%20Store%20-%20GVS%20Quickstart).  Make special note of the "Prerequisites" section.
+3. Run the workflows outlined in the Quickstart description in the workspace that contains the data by using the "Copy to Another Workspace" function.  The first two steps (GvsAssignIds and GvsImportGenomes) can be run multiple times to load all necessary sample sets into one "instance" of GVS.  The rest of the steps are only run once, in order, and only after all the samples for the callset are loaded.
+   1. GvsAssignIds at the sample set level ("Step 2")
+   2. GvsImportGenomes at the sample set level ("Step 2")
+   3. GvsCreateAltAllele
+   4. GvsCreateFilterSet
+      - make note of the “filter_set_name” input used
+   5. GvsPrepareCallset
+      - if the callset has more than 20K genomes, you will want to reserve GCP flex slots for this step and then release them once it has completed
+   6. GvsExtractCallset
+       - include a valid (and secure) “output_gcs_dir” parameter, this is where the VCF and interval list files  will go
+       - make note of the “cohort_extract_table_prefix” input that you enter
+   7. Run [callset QC](callset_QC/README.md), for which you need
+      1. permission to query table `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites`
+      2. the data_project you used for all the GVS WDLs
+      3. the default_dataset you used for all the GVS WDLs
+      4. the “cohort_extract_table_prefix” input from `GvsExtractCallset` step
+      5. the “filter_set_name” input from `GvsCreateFilterSet` step
+
+## Deliverables
+1. location of the VCF and interval_list files (“output_gcs_dir” input from GvsExtractCallset)
```

Comment:
Do we also deliver the BQ tables to support cohort extraction?

Reply:
we do share the BQ dataset as well as the gs path to the sharded vcfs with the workbench team